### ISSUE TITLE
Failed upgrade from OpenERP 7 to Odoo 8.0.1.3 - "ImportError: No module named openerplib"

### ISSUE THREAD

**Issue by ViktorNova** [OCA/OpenUpgrade#178]
On a Debian 7 server, I first upgraded to Odoo through the package manager, then ran the migration script. No errors on the console, but Odoo failed to start, complaining the database wasn't the correct version. Looking at the migration log, the errors all boil down to one:

`ImportError: No module named openerplib`

The whole log, plus a few more details are on a previous issue I posted (I closed it because I think I've nailed down the core problem, and the original topic was ambiguous) https://github.com/OpenUpgrade/OpenUpgrade/issues/156

What should I do?

**Comment by clikinfo** [comment]
Same problem :(

**Comment by clikinfo** [comment]
sudo easy_install openerp-client-lib

**Comment by ViktorNova** [comment]
You are a life saver, thank you!

I had to run it a few more times, checking the migration log each time to see what failed, and ended up having to install a few additional python packages as well (i used pip instead of easy_install)

```
sudo pip install openerp-client-lib
sudo pip install decorator
sudo pip install requests
sudo pip install pyPdf
sudo pip install passlib
```

hope this helps someone else!

**Comment by StefanRijnhart** [comment]
For reference, fixed for 8.0 by @pedrobaeza in #181
